### PR TITLE
xtest storage: add a test case of creation of already existing object

### PR DESCRIPTION
**Pull request by etienne-lms**
Regression xtest 6012 already tests creation and recreation of a persistent object. This change adds a test in 6012 to re-create the existing object without setting the overwrite attribute. Creation should fail but the object should not be altered and former open should succeed.

### CODE REVIEW

**Comment by etienne-lms** [comment]
This change is related to issue https://github.com/OP-TEE/optee_test/issues/232 and fix https://github.com/OP-TEE/optee_os/pull/1903.

File: host/xtest/regression_6000.c [hunk 1]
```diff
@@ -1406,43 +1406,68 @@ static void xtest_tee_test_6012_single(ADBG_Case_t *c, uint32_t storage_id)
...
 	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
 		fs_create_overwrite(&sess, file_04, sizeof(file_04), storage_id)))
-		goto exit;
+		goto bail1;
 
 	TEEC_CloseSession(&sess);
 
-	/* re-create the same */
+	/*
+	 * re-create the object two times with overwrite attribute
```
Comment (jforissier) [inline host/xtest/regression_6000.c]: You mean a second time?

Comment (etienne-lms) [reply host/xtest/regression_6000.c]: The sequence below re-creates twice the already existing object. I left it as it was.
Maybe I could say `re-create twice again the object with overwrite attribute`?

Comment (jforissier) [reply host/xtest/regression_6000.c]: No it's OK

**Comment by etienne-lms** [comment]
tag applied.

**Pull request merged**